scripts/data/getBoundingBox.yml

```diff
@@ -1,43 +1,29 @@
 script: getBoundingBox.R
 name: Get bounding box
-description: "Exrtact the bounding box of a country or region of interest."
+description: Extract the bounding box from a polygon of interest (to choose a country, connect to theGet Country Polygon script).
+license: CC-BY
 author:
   - name: Jory Griffith
     identifier: https://orcid.org/0000-0001-6020-6690
 inputs:
-  country:
-    label: Country
-    description: Country of interest. For list of countries look here https://geoio.biodiversite-quebec.ca/docs#/default/validate_country_name_validate_country_name_get.
-    type: text
-    example: Canada
-  state:
-    label: State/Province
-    description: State or province of interest (leave blank if you want the full country). For list of accepted region names look here https://geoio.biodiversite-quebec.ca/docs#/default/validate_state_name_validate_state_name_get.
-    type: text
-    example: Quebec
   study_area_file:
-    label: File for study area
-    description: Upload a file of a custom study area
-    type: text
+    label: Polygon of study area
+    description: Upload a file of a custom study area in geopackage format
```

Comment:
This should say: "Local path or url linking to file of custom study area, preferably in geopackage format containing only one layer."